lib/os-cluster-entrypoint.ts

```diff
@@ -84,7 +84,13 @@ export class OsClusterEntrypoint {
       if (securityDisabled !== 'true' && securityDisabled !== 'false') {
         throw new Error('securityEnabled parameter is required to be set as - true or false');
       }
-      const security = securityDisabled === 'true';
+      const insecure = securityDisabled === 'true';
+
+      const adminPassword: String = insecure ? '' : `${scope.node.tryGetContext('adminPassword')}`;
+      console.log(`Admin password is ${adminPassword}`);
```

Comment:
Debug console statement? Printing password to the console can be risky esp when it comes to deployments.

Reply:
Added it for testing, missed to remove it. Removed.